ChangeLog.d/cmake_add_subdirectory_support.txt

```diff
@@ -0,0 +1,4 @@
+Changes
+   * Add aliases for libraries so that the normal MbedTLS::* targets
```

Comment:
Please indicate clearly near the beginning that this is about CMake. For readers that don't have CMake in mind, the first sentence looks quite mysterious.

Reply:
Done in https://github.com/Mbed-TLS/mbedtls/pull/5905.